* always use OpenSSL's MD5, never Solaris' MD5 implementation (we want
  to use whatever we provide ourself, but also not get libintl.so.1
  references from non-prefix dragged in...)

--- configure.in
+++ configure.in
@@ -311,27 +311,6 @@
   MD5_OBJ='gen-md5$o'
   found_md5=no
 
-  dnl Check for the system MD5 library on Solaris.  We don't check for
-  dnl something simple like "MD5Update" because there are a number of
-  dnl MD5 implementations that use that name, but have an otherwise
-  dnl incompatible interface.  md5_calc is, hopefully, specific to the
-  dnl Solaris MD5 library.
-  if test x"$found_md5" = xno; then
-    AC_CHECK_LIB(md5, md5_calc, [
-      dnl Some installations have bogus <md5.h> in the compiler's
-      dnl include path, making the system md5 library useless.
-      AC_MSG_CHECKING([for working md5.h])
-      AC_COMPILE_IFELSE([#include <md5.h>
-                        ], [
-        AC_MSG_RESULT(yes)
-        AC_DEFINE([HAVE_SOLARIS_MD5], 1, [Define to use Solaris MD5.])
-        LIBS="-lmd5 $LIBS"
-        found_md5=yes
-        AC_MSG_NOTICE([using the Solaris MD5 implementation])
-      ], [AC_MSG_RESULT(no)])
-    ])
-  fi
-
   dnl Then see if we're linking OpenSSL anyway; if yes, use its md5
   dnl implementation.
   if test x"$found_md5" = xno; then
